### DISCUSSION TITLE
Listing open sources of geographic information that can be easily imported into UMEP

### DISCUSSION THREAD

**Discussion by oraleks** [UMEP-dev/UMEP#360] [Ideas]
WUDAPT connectivity with UMEP can and should be enhanced to enable users to exploit its growing database of global urban geographic data. Since the availability of open geographic data is essential to UMEP usability, an online list that documents open sources that can be used for the task may be useful if integrated into the UMEP GitHub site.

**Comment by biglimp** [comment]
I agree. However, I wonder if not such a list already exists somewhere.

**Comment by rarygit** [comment]
You could look at this another way.

There are large metropolitan cities which already provide open data, along with federal or nationwide open spatial data. 
For example, point clouds, orthophotography, building footprints etc. An internet search will find these quite easily.
In some circumstances the dataset sizes range from 2 Gb to 12 Gb; you might need six datasets. 
You might need to download 32 Gb to preprocess your site. At 2 mb/s download, it can be done.
In the category of unpleasant surprises, the Australian 5 m DEM is 6 Gb to download and then extracts itself to 270 Gb.

However I would also suggest that there is a more urgent case for a database that list tools and techniques to create missing spatial data.
Furthermore, uploading your newly minted spatial data, using said technique, to an online repository so that others can use it, augment it, or repreduce your research. At the very least, you could upload guidance on how you used the technique.

For example:
How to Create a DEM using Sentinel-1 Data
https://asf.alaska.edu/how-to/data-recipes/create-a-dem-using-sentinel-1-data/

How to create a DSM using Sentinel-1 data
Lo-Key Search: [how to create a DSM using Sentinel-1](https://duckduckgo.com/?q=how+to+create+a+DSM+using+Sentinel-1&t=chromentp&ia=web)

(Some) Search Results:

Validation of Sentinel-1A and AW3D30 DSMs for the Metropolitan Area of Istanbul, Turkey
http://www.sefercik.com/wp-content/uploads/2017/05/Sefercik2018_Article_ValidationOfSentinel-1AAndAW3D.pdf

Preliminary results of using Sentinel-1 SAR data for DSM generation
https://www.researchgate.net/publication/298083311_Preliminary_results_of_using_Sentinel-1_SAR_data_for_DSM_generation

Get DSM by sentinel-1 SLC
https://forum.step.esa.int/t/get-dsm-by-sentinel-1-slc/10933
